docs/languages/_posts/2019-03-04-visual-basic.md

```diff
@@ -0,0 +1,7 @@
+According to Wikipedia, Visual Basic .NET (VB.NET) is a programming language that was created by Microsoft in 2001. Beyond that, Wikipedia doesn’t have much to offer in terms of historical information, so let’s move on.
+
+In terms of features, however, Wikipedia has a little more to offer. For instance, VB.NET is statically typed, but the type system is both strong and weak as well as safe and unsafe.
+
+As for syntax, VB.NET is structured which reminds me of languages like Pascal and MATLAB where blocks of code are marked by keywords—rather than braces or whitespace.
+
+Beyond that, VB.NET has several versions. In fact, VB.NET only really refers to the language before 2005, but the syntax is largely the same today. Of course, I’ve sort of lumped them together in the same way I’ve recognized Python as one language instead of two.
```

Comment:
This should be changed from `Of course, I’ve sort of lumped them together in the same way I’ve recognized Python as one language instead of two.` to something like `These have been lumped  together, much in the same way Python is recognized as one language rather than two.`

Reply:
At least in my opinion

Reply:
Good point! Keep in mind this is using github’s new draft pull request feature. I was just in the process of dumping some old content over, so they’re not ready to go yet.